TypeScript error: createLinkManager called with wrong number of arguments

## Description

`src/isolated-renderer/widget-provider.tsx` calls `createLinkManager` with only 1 argument, but the function signature requires 2:

```typescript
// widget-provider.tsx line 95
useEffect(() => createLinkManager(client.store), [client.store]);
```

```typescript
// link-subscriptions.ts
export function createLinkManager(
  store: WidgetStore,
  sendUpdate: SendUpdate,  // Missing in call
): () => void {
```

## Error

```
src/isolated-renderer/widget-provider.tsx(95,19): error TS2554: Expected 2 arguments, but got 1.
```

## Fix

Pass `client.sendUpdate` as the second argument:

```typescript
useEffect(() => createLinkManager(client.store, client.sendUpdate), [client.store, client.sendUpdate]);
```

## Impact

This breaks TypeScript compilation (`npx tsc --noEmit` fails).

--- a/src/isolated-renderer/widget-provider.tsx
+++ b/src/isolated-renderer/widget-provider.tsx
@@ -92,7 +92,10 @@ export function IframeWidgetStoreProvider({
   }, [client]);
 
   // Set up link subscriptions (jslink/jsdlink)
-  useEffect(() => createLinkManager(client.store), [client.store]);
+  useEffect(
+    () => createLinkManager(client.store, client.sendUpdate),
+    [client.store, client.sendUpdate],
+  );
 
   // Set up canvas manager router (ipycanvas)
   useEffect(() => createCanvasManagerRouter(client.store), [client.store]);

